src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -330,6 +333,8 @@ public async Task ProcessInputAsync(Message message, Flow flow, IContext message
             }
             catch (Exception ex)
             {
+                ex = _analyzeBuilderExceptions.VerifyFlowConstructionException(ex);
+
                 if (inputTrace != null)
                 {
                     inputTrace.Error = ex.ToString();
```

Comment:
in line 344, replace "Error processing input with ID" to "Error processing input with message ID"

Reply:
Done